velox/functions/lib/JodaDateTime.cpp

```diff
@@ -188,105 +182,109 @@ parseTimezoneOffset(const char* cur, const char* end, JodaDate& jodaDate) {
   throw std::runtime_error("Unable to parse timezone offset id.");
 }
 
-} // namespace
+void parseFromPattern(
```

Comment:
This function is copied from a subset of `parse` function below so no need to review. Easier to see the change on Phabricator for this one.